include/triton/backend/backend_input_collector.h

```diff
@@ -232,8 +235,10 @@ class BackendInputCollector {
   const bool pinned_enabled_;
   const bool use_async_cpu_copy_;
   cudaStream_t stream_;
```

Comment:
Wouldn't stream_ run into similar error?

Reply:
`stream_` is ok, because it is referenced at multiple places regardless of the `TRITON_ENABLE_GPU` setting:

https://github.com/triton-inference-server/backend/blob/main/src/backend_input_collector.cc#L379
https://github.com/triton-inference-server/backend/blob/main/src/backend_input_collector.cc#L521
https://github.com/triton-inference-server/backend/blob/main/src/backend_input_collector.cc#L618
https://github.com/triton-inference-server/backend/blob/main/src/backend_input_collector.cc#L697
https://github.com/triton-inference-server/backend/blob/main/src/backend_input_collector.cc#L980